src/Microsoft.Dnx.Tooling/Restore/RestoreOperations.cs

```diff
@@ -46,8 +87,9 @@ public RestoreOperations(IReport report)
                     }
                 }
 
+                dependencies.AddRange(node.Item.Dependencies);
+
                 var tasks = new List<Task<GraphNode>>();
-                var dependencies = node.Item.Dependencies ?? Enumerable.Empty<LibraryDependency>();
```

Comment:
Not enough old code is removed. It's still evaluating runtime specs for each dependency and that's unnecessary.